docs/api/source/conf.py

```diff
@@ -3,17 +3,15 @@
 import os
 import pkgutil
 import sys
-from dataclasses import dataclass
-from dataclasses import field
-from typing import Any, Dict, List
+from typing import Any, Dict
 
-from sphinx.ext.autodoc import mock
```

Comment:
@vshampor should we skip pylint message here or should we do something with that?